workflows/cloudify_system_workflows/snapshot.py

```diff
@@ -571,13 +622,19 @@ def recreate_deployments_environments():
 
 
 @workflow(system_wide=True)
-def restore(snapshot_id, recreate_deployments_envs, config, **kwargs):
+def restore(snapshot_id, recreate_deployments_envs, config, force, **kwargs):
     mappings = {
         '3.3': _restore_snapshot_format_3_3,
         '3.2': _restore_snapshot_format_3_2
     }
 
     config = _DictToAttributes(config)
+
+    es = _create_es_client(config)
+
+    if not force:
```

Comment:
I think it would be a good idea to check whether the manager is clean anyway, and have a warning log regardless of whether force=true was passed or not. The force=true will skip the validation but the warning will still be there.